test/unit/specs/http_connector.js

```diff
@@ -434,7 +434,7 @@ describe('Http Connector', function () {
       con.request({
         body: body
       }, function (err, resp, status) {
-        expect(http.ClientRequest.prototype.setHeader.lastCall.args).to.eql(['Content-Length', 14]);
+        expect(http.ClientRequest.prototype.setHeader.args.find((arg) => arg[0] === 'Content-Length')).to.eql(['Content-Length', 14]);
```

Comment:
`nock@9` seems to add the `host` header, so `content-length` is no longer the `lastCall`.